content/docs/replication/replication-kubevirt.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Replication with KubeVirt
+linktitle: Replication with KubeVirt
+weight: 6
+description: >
+  Support of Replication enabled CSI drivers with virtualized workloads.
+---
+
+Starting in v1.11.0, the Replication module supports the volume replication for
+Persistent Volumes (PVs) created via virtualized workloads (VMs).
+
+## KubeVirt:
+
+- KubeVirt allows virtual machines (VMs) to run alongside container workloads
+  within Kubernetes clusters, addressing the coexistence of legacy VM workloads
+  and modern containers.
+
+## KubeVirt support with Replication
+
+- Install the OpenShift Virtualization operator from the RedHat UI on both the
+  source and target clusters with OCP v4.18 and above.
+- The Replication modes supported are: SYNC and ASYNC.
+
+- Make sure the driver is installed with the Replication Module enabled and a
+  default storage class configured for replication. To set the replication
+  storage class as the default, update the annotation as follows:
+
+  ```
+  kubectl patch storageclass replication-storageclass -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+  ```
+
+- Deploy a VM on the source cluster to provision a Persistent Volume (PV) using
+  a data volume template or a Persistent Volume Claim (PVC). The PV created on
+  the source will be replicated to the target cluster, as there is an active
+  replication session between the two clusters.
+
+- sample _create-vm.yaml_ manifest:

```

Comment:
Point to RedHat documentation here instead

Reply:
Addressed.